components/nexusoperations/workflow/commands.go

```diff
@@ -216,27 +216,22 @@ func (ch *commandHandler) HandleCancelCommand(
 		if errors.Is(err, hsm.ErrStateMachineNotFound) {
 			return workflow.FailWorkflowTaskError{
 				Cause:   enumspb.WORKFLOW_TASK_FAILED_CAUSE_BAD_REQUEST_CANCEL_NEXUS_OPERATION_ATTRIBUTES,
-				Message: fmt.Sprintf("requested cancelation for a non-existing operation with scheduled event ID of %d", attrs.ScheduledEventId),
-				// TODO(bergundy): Message: fmt.Sprintf("requested cancelation for a non-existing or already completed operation with scheduled event ID of %d", attrs.ScheduledEventId),
+				Message: fmt.Sprintf("requested cancelation for a nonexistent or terminated operation with scheduled event ID %d", attrs.ScheduledEventId),
```

Comment:
The message I had in the todo is better. The operation isn't terminated.

Reply:
Done